api/projects.py

```diff
@@ -119,7 +119,7 @@ def get(self, uid=None, gid=None):
             if not group:
                 self.abort(400, 'invalid group id')
         query = {'group': gid} if gid else {}
-        projection = ['group', 'name', 'notes', 'timestamp', 'timezone']
+        projection = ['group', 'name', 'notes', 'timestamp', 'timezone', 'public']
```

Comment:
I was going for the minimal change set to achieve the requested behavior. For consistency, we may indeed want to update the projections of all containers.